python-modules/cis_aws/tests/test_connect.py

```diff
@@ -463,4 +463,5 @@ def test_pyconfig_support(self):
         assert test_port == '34567'
 
     def teardown(self):
```

Comment:
just a note, i think `__delete__` would be the pythonic way (it also trigger on object being deleted automatically)